js1/funkce-obory/null-undefined.md

```diff
@@ -47,46 +55,38 @@ Pokud to v programu potřebujeme, můžeme proměnnou na hodnotu `null` otestova
 
 ```js
 if (msgElm === null) {
-  console.log('Element nenalezen');
+  document.body.innerHTML = 'Element nenalezen';
```

Comment:
Záměrně tu není `+=`?

Reply:
Ano, tady a v dalších případech jsem to dal záměrně jako přepis stránky. Pokud ten kód budou mít v jednom souboru, hromadilo by se jim na stránce různé 'Element nenalezen' a 'undefined' a nebudou vědět, co k čemu patří. Tohle je taková snaha donutit je vyzkoušet vždy jen jednu věc.

Reply:
Takhle si zase přepíšou všechny chyby a nebudou vědět, že jich mají více, ne?

Asi teda klidně nech takto, ale já v „hromadění“ nevidím problém.

Reply:
Jednou někdo z lektorů zapomněl `+=` a nějaká účastnice ho na to upozornila. Já bych byl spíš pro konzistenci. 🤷‍♀️

Reply:
Upravil jsem to všude na `+=`. Aspoň se tím vyřešil ten problém, že `innerHTML = null` stránku jen vymaže, nezobrazí se tam `null`. `innerHTML += null` do stránky přidá text „null“.

Reply:
Cool 👀